SensioLabs/Security/Crawler.php

```diff
@@ -97,6 +97,19 @@ private function doCheck($lock, $format = 'json', array $contextualHeaders = [])
             ],
         ];
 
+        $proxy = getenv('HTTPS_PROXY');
+        if ($proxy) {
+            $parts = parse_url($proxy);
+            if (!is_array($parts) || (!isset($parts['path']) && !isset($parts['host']))) {
+                throw new RuntimeException(sprintf('Failed to parse HTTPS_PROXY "%s".', $proxy));
+            }
+            $opts['http']['proxy'] = sprintf(
+                '%s:%d',
+                isset($parts['host']) ? $parts['host'] : $parts['path'],
```

Comment:
```suggestion
                $parts['host'] ?? $parts['path'],
```

Reply:
`??` wasnt'added untill php 7.0, and this project supports php versions lower than that, so it can not be used.

Reply:
Oops as much for me 😅